### DISCUSSION TITLE
Group by and sum fields in an implicit many to many relationship

### DISCUSSION THREAD

**Discussion by jeepers3327** [prisma/prisma#10375] [Q&A] [answered]
Hi, How do I group and sum records within Implicit Many to Many.
Currently I have a working prisma query but I wonder if there's another way in it.

```
  const countTags = await ctx.prisma.tag.findMany({
    select: {
      name: true,
      _count: {
        select: {
          questions: true,
        },
      },
    },
  });
```

Since its implicit I cannot call the third table to query it directly.
Here's my schema:
```
model Question {
  id        Int      @id @default(autoincrement())
  title     String
  content   String
  tags      Tag[]
}

model Tag {
  id   Int    @id @default(autoincrement())
  name String @unique
  questions Question[]
}
```
I expect the output is like this (although the exact shape of the object might be different)
```
[
 {name: 'SomeTag', count: 3}, 
 {name: 'SampleTag', count: 4},
 {name: 'SampleTag', count: 1}
]
```

**Comment by austincrim** [comment, marked as answer]
Hey @jeepers3327 👋, your original query seems good to me!

If you have other ideas about how you would like to query this data, feel free to open a [feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) with more details.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
